css/style007.css

```diff
@@ -0,0 +1,89 @@
+.manju{
+padding-bottom:60px;
+font-family: Titillium Web;
+}
+
+.header{
+ font-size:50px;
+
+}
+
+.imge {
+	padding-bottom: 10rem;
+    width: 100%;
+    height: 40%;
+    
+}
+
+
+.manz{
+  padding-right:50px;
+  padding:10px;
+  font-family:arial;
+  font-size:3rem;
+  background-color:#166ad1;
+  color:white;
+
+}
+
+
+.set{
+  background-color:white;
+}
+
+
+/* carousel */
+.media-carousel 
+{
+  margin-bottom: 0;
+  padding: 0 40px 30px 40px;
+  margin-top: 30px;
+}
+/* Previous button  */
+.media-carousel .carousel-control.left 
```

Comment:
DO NOT use predefined classes from bootstrap. Change these class names and update. 
Addional to that, please use class prefixes to all class names.